app/actions/GroupActions.js

```diff
@@ -84,6 +84,19 @@ export function fetchAllWithType(type: string): Thunk<any> {
   });
 }
 
+export function fetchRandomInterestgroups(): Thunk<any> {
```

Comment:
```suggestion
export function fetchRandomInterestGroups(): Thunk<any> {
```
For consistency interest groups should be written as two words. In `selectRandomInterestgroups` also.